cwl/workflows/parts/1_mgygs/1.cwl

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env cwl-runner
+cwlVersion: v1.2
+class: Workflow
+
+requirements:
+  SubworkflowFeatureRequirement: {}
+  MultipleInputFeatureRequirement: {}
+  InlineJavascriptRequirement: {}
+  StepInputExpressionRequirement: {}
+  ScatterFeatureRequirement: {}
+
+inputs:
+  genomes_ena: Directory?
+  ena_csv: File?
+  genomes_ncbi: Directory?
+
+  max_accession_mgyg: int
+  min_accession_mgyg: int
+
+  # skip dRep step if MAGs were already dereplicated
+  skip_drep_step: string   # set "skip" for skipping
+
+  # no gtdbtk
+  skip_gtdbtk_step: string   # set "skip" for skipping
+
+  # common input
+  mmseqs_limit_c: float
+  mmseqs_limit_i: float[]
+  mmseq_limit_annotation: float
+
+  gunc_db_path: File
+
+  gtdbtk_data: Directory?
+
+  InterProScan_databases: [string, Directory]
```

Comment:
I think using lowercase everywhere make things simpler, mostly aesthetic change, but it helps readability IMO